requirements/common.txt

```diff
@@ -5,7 +5,6 @@ Flask-SQLAlchemy>=2.1,<2.2
 Flask-Migrate>=2.0.0,<3
 Flask-Login>=0.4,<1
 Flask-Scrypt>=0.1.3.6,<0.2
-flask-restplus>=0.9.2,<1
 Flask-JWT>=0.3.2,<0.4
 eventlet>=0.20.0,<1
```

Comment:
Can be removed if `Flask-SocketIO` is removed. This was required only for that, but we have anyway stopped using this since a few months back.